### ISSUE TITLE
Extend documentation to list permissions (on transport) needed to operate

### ISSUE THREAD

**Issue by Floopy-Doo** [Particular/ServiceControl#3406]
We are currently setting up service control (Error, Audit and Monitoring) in our environment. We are using Azure Service Bus and our security team is keen on restricting access as much as possible to the bus.

We had to use a lot of trial and error to figure out which instance need which permissions on which queue / topic to operate properly and are still not sure if we caught all use cases.

I would propose to add a section to the documentation that lists the permissions required on the bus.
Maybe its already in the documentation and we haven't found it, yet. 

Kind regards
Florian

**Comment by kbaley** [comment]
This is a good idea though it would be a bit of an undertaking to keep it up-to-date for each transport. We'll keep this in mind when we do our next documentation update.

**Comment by Floopy-Doo** [comment]
Some intel we gathered during our endeavor with ASB (Azure Service Bus), RBAC (Role-based Access Control) and MI (Managed Identities). 

We ended up giving ServiceControl the role "_Azure Service Bus Data Owner_" on all queues and topic 
and a custom role on the namespace that consisted only of the single claim "_Microsoft.ServiceBus/namespaces/queues/read_".

The permissions on queues and topics are needed so messages can be ingested from the error/audit queues and to allow retry on the FailedQ queues, as well as publishing ServiceControl messages to the topic and working with the ServiceControl internal queues. The namespace claim is only needed by the monitoring instance to monitor queue lengths (for ref. QueueLengthProvider.cs). 

We assume we could get away with "_Azure Service Bus Data Receiver_" and "_Azure Service Bus Data Sender_" roles on each queue/topic and the custom role with the claim "_Microsoft.ServiceBus/namespaces/queues/read_" on the whole namespace. We decided to stay with the owner for now.

I hope this helps.
